src/process/enumeration.rs

```diff
@@ -0,0 +1,85 @@
+use super::{Enumeration, EnumerationMember};
+use log::warn;
+
+use crate::api_map::APIMap;
+
+/// Convert a snake_case or SCREAMING_SNAKE_CASE string to PascalCase
```

Comment:
IMO it's more readable to name it `make_pascal_case` or `convert_to_pascal_case` or something